addons/hypothermia/stringtable.xml

```diff
@@ -136,6 +150,7 @@
             <Dutch>Verwarmt IV-vloeistoffen. Moet geplaatst worden op een al aangelegde lijn.</Dutch>
             <Spanish>Provee calor a los fluidos IV. Deben ser colocados en una línea ya existentes</Spanish>
             <Japanese>IVの輸液を温めます。既存のライン上に配置する必要があります</Japanese>
+            <German>Erwärmt IV-Flüssigkeiten. Muss auf einer bereits vorhandenen Zeile platziert werden</German>
```

Comment:
"Erwärmt IV-Flüssigkeiten. Muss auf einer bereits vorhandenen Leitung platziert werden" would be better